matchbox/resource_profile.go

```diff
@@ -57,9 +63,13 @@ func resourceProfileCreate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*matchbox.Client)
 	ctx := context.TODO()
 
+	if d.HasChange("raw_ignition") && d.HasChange("container_linux_config") {
```

Comment:
Hm, is it possible to get in a state where both are defined but both don't have changes? We should error if both are non-empty.